### DISCUSSION TITLE
Facebook lite assistance

### DISCUSSION THREAD

**Discussion by gothmog123** [ImranR98/Obtainium#1522] [Q&A]
FB's lite apk is available for download on their html site:

https://m.facebook.com/lite/

Any chance somebody could help me get it with obtainium?

**Comment by DwainZwerg** [comment]
Hello @gothmog123 ,
If I have JavaScript enabled in my browser, I can find both links. Both the Play Store link and the APK link. As soon as I deactivate JavaScript, the APK link disappears. So there is nothing we can do.

**Comment by Julmik647** [comment]
I think that Facebook lite similary to other Meta apps updates automatically through Meta app manager that installs when you install Meta app.
